docs/common/install_and_configure_nginx_with_rtmp_module_on_windows.md

```diff
@@ -60,12 +107,95 @@ stream {
         proxy_pass live;
         ssl_certificate c:\\certs\\fullchain.pem; #root path where your certificate is located e.g.: C:\certs\fullchain.pem
         ssl_certificate_key c:\\certs\\privkey.pem; #root path where your certificate key is located e.g.: C:\certs\privkey.pem
-
         allow all;              # this is public (this is also the default)
     }
 }
 
 rtmp {
+    server { 
+        listen 127.0.0.1:2951;
+        chunk_size 4096;
+
+        application secure-extraction {
+            live on;
+            record off;
+            allow publish 127.0.0.1;
+        }
+    }
+
+    server { 
+        listen 127.0.0.1:2952;
+        chunk_size 4096;
+
+        application secure-extraction {
+            live on;
+            record off;
+            allow publish 127.0.0.1;
+        }
+    }
+
+    server { 
+        listen 127.0.0.1:2953;
+        chunk_size 4096;
+
+        application secure-extraction {
+            live on;
+            record off;
+            allow publish 127.0.0.1;
+        }
+    }
+
+   server { 
+        listen 127.0.0.1:2954;
+        chunk_size 4096;
+
+        application secure-extraction {
+            live on;
+            record off;
+            allow publish 127.0.0.1;
+        }
+    }
+
+   server { 
+        listen 2941;
```

Comment:
**nitpick(blocking)**: For RTMP we are using 1xxx PORTS, and 2XXX for RTMPS. We must be consistent